bindings/src/main/java/io/pravega/storage/filesystem/FileSystemStorageConfig.java

```diff
@@ -28,7 +28,8 @@
 @Slf4j
 public class FileSystemStorageConfig {
     //region Config Names
-
+    public static final Property<Integer> WRITE_CACHE_SIZE = Property.named("cache.write.size", 1024);
```

Comment:
When we talk to the file system, The configuration `cache.write.size` will make users think of the **SPACE**  of cache, instead of the **COUNT** of the file channel. After all, many people know the file system has a page cache. If this configuration only works for the file system, I think the `filesystem.cache.write.channel.size` or something else is better.

Reply:
If you agree on the above, I think the READ_CACHE_SIZE could be renamed to `READ_CHANNEL_CACHE_SIZE `